platform/admin/admin-configuration-configinstaller/src/main/java/org/codice/ddf/admin/configuration/ConfigurationInstaller.java

```diff
@@ -171,7 +214,7 @@ private void handleUpdate(ConfigurationEvent event) {
     private void handleDelete(ConfigurationEvent event) {
         String pid = event.getPid();
         File felixFileOriginal = pidFileMap.remove(pid);
```

Comment:
❓ Is it possible that the map and file system get out of sync if performOperation() fails?